scheduler/src/cook/mesos/api.clj

```diff
@@ -1015,39 +1015,72 @@
 ;;
 
 (def ReadRetriesRequest
-  {:job s/Uuid})
+  {(s/optional-key :job) s/Uuid
+   (s/optional-key :jobs) [s/Uuid]})
 (def UpdateRetriesRequest
-  (merge ReadRetriesRequest {:retries PosNum}))
+  (merge ReadRetriesRequest {(s/optional-key :retries) PosNum
+                             (s/optional-key :increment) PosNum}))
 
 (defn display-retries
   [conn ctx]
-  (:job/max-retries (d/entity (db conn) [:job/uuid (::job ctx)])))
+  (:job/max-retries (d/entity (db conn) [:job/uuid (-> ctx ::jobs first)])))
 
-(defn check-job-exists
+(defn jobs-from-request
+  [ctx]
+  (let [single-job (or (get-in ctx [:request :query-params :job])
```

Comment:
@wyegelwel you're referring to the repeated job query parameters when requesting the status of multiple jobs.   That is one way of obtaining an array-like data structure from query parameters "&job=x&job=y".  It's not a strategy that's really applicable to this here, because we're talking about the JSON payload of a POST request.   You can't repeat the same key in a JSON hash... e.g {"foo": 1, "foo": 2} isn't valid JSON.  

So, there were a few choices:

1. make it so that "job" parameter can be either a string or an array, either "job": "uuid" or "job": ["uuid1", "uuid2"]
2. get rid of the "job" parameter and only support "jobs" - so only "jobs": ["uuid1", "uuid2"]
3. Just allow either parameter, {"job": "uuid1"} or  {"jobs": ["uuid1", "uuid2"]}

My thoughts were: #1  makes for kind of a weird, non-specific schema.  #2 would break backwards compatibility.